llvm/lib/Transforms/Vectorize/LoopVectorizationLegality.cpp

```diff
@@ -1441,10 +1441,12 @@ bool LoopVectorizationLegality::canVectorize(bool UseVPlanNativePath) {
   // Check whether the loop-related control flow in the loop nest is expected by
   // vectorizer.
   if (!canVectorizeLoopNestCFG(TheLoop, UseVPlanNativePath)) {
-    if (DoExtraAnalysis)
+    if (DoExtraAnalysis) {
+      LLVM_DEBUG(dbgs() << "LV legality check failed: loop nest");
```

Comment:
Perhaps possible to check this message as well? Shouldn't the debug message start with an "LV: " prefix for uniformity?

Reply:
Done